### ISSUE TITLE
Minor Visual Bug on Preferences Icon

### ISSUE THREAD

**Issue by Zenohm** [Ulauncher/Ulauncher#107]
## Problem

While using ULauncher I noted a small visual error in the preferences icon. Notably, the icon had a slight extension on the right side causing it to form an almost USS Enterprise-esque shape. This error disappears if I hover over the icon, however it does not disappear if I change to light mode, or hide recent results.

## Pictures

- Visual Bug Present:

![Bug Present](http://i.imgur.com/ULt21Yp.png)

- Visual Bug Absent:

![Bug Absent](http://i.imgur.com/HijihWE.png)

## General System Information (via Neofetch):

```
OS: Arch Linux x86_64 
Model: Aspire V5-552 V2.02 
Kernel: 4.9.13-1-lts 
Uptime: 10 hours, 54 mins 
Packages: 1951 
Shell: zsh 5.3.1 
Resolution: 1920x1080 
DE: Xfce 
WM: Xfwm4 
WM Theme: OSX-Arc-Darker 
Theme: OSX-Arc-Darker [GTK2], Adapta-Eta [GTK3] 
Icons: La-capitaine-icon-theme [GTK2], Paper [GTK3] 
Terminal: termite 
Terminal Font: Consolas 12 
CPU: AMD A10-5757M APU (4) @ 2.5GHz 
GPU: AMD Radeon HD 8650G 
Memory: 1945MiB / 5414MiB 
```

If more information in a specific system is required, I'd be happy to provide that. I haven't yet tested to see if this error exists in other distributions or with other system configurations.
Note that I am running ULauncher v2.0.30.r3.

**Comment by Zenohm** [comment]
I did a bit of research going through some of the old closed issues searching for pictures, and I have noticed a slight trend. From what I've seen by looking through these examples, this bug is only present on non-Ubuntu (and family) distributions. This bug specifically seems to be present primarily on Arch Linux, although it appears as though a similar bug is or at least was present on other distributions (i.e. Fedora 24) in the past.

## Visual Bug Present

- In #48 there was a [picture posted by Cross03 who was running Arch Linux.][1]
- In #87 there was a [picture posted by the creator, gornostal, who was running Fedora 24 in a virtual machine.][2] 
..- The bug here is slightly different in that it has been rotated 90 degrees clockwise, and it was also present when the icon was being hovered over.
..- This issue appears to have been addressed in #87. Perhaps the solution used there can indicate a solution here.
- In this issue posting, I posted a [picture of ULauncher running on Arch Linux.][3]


## Visual Bug Absent

- In #58 there was a [gif posted by fabubaker who appears to have been running Ubuntu.][4]
- In #60 there was a [picture posted by hughwilliams94 who was running GalliumOS.][5]
..- GalliumOS is based on XUbuntu, which is a flavor of Ubuntu.
- In #104 there was a [picture posted by orschiro who was running Ubuntu.][6]
- On the website there is a [video demo of ULauncher V2.0 running on what appears to be Ubuntu.][7]

[1]: https://github.com/Ulauncher/Ulauncher/issues/48#issuecomment-280985826
[2]: https://github.com/Ulauncher/Ulauncher/issues/88#issue-210331806
[3]: https://github.com/Ulauncher/Ulauncher/issues/107#issue-213616020
[4]: https://github.com/Ulauncher/Ulauncher/issues/58#issuecomment-280948536
[5]: https://github.com/Ulauncher/Ulauncher/issues/60#issue-208841271
[6]: https://github.com/Ulauncher/Ulauncher/issues/104#issue-213267587
[7]: https://www.youtube.com/watch?v=LDkxqgtHPGk

**Comment by gornostal** [comment]
Wow! Thanks for an exemplary bug report @Zenohm.
I'll look into it

**Comment by gornostal** [comment]
@Zenohm 
When you run ulauncher from terminal like this `ulauncher -v` it logs GTK+ version.
Can you tell what it is on your machine?

Log entry will look something like this:
`2017-03-23 22:14:53,775:INFO:ulauncher: main() 'GTK+ 3.20.9'`

**Comment by gornostal** [comment]
I tried on my Archlinux in VBox and I haven't seen that issue.
GTK+ 3.22.10 is installed there.
Although I'm using KDE, I don't think that has anything to do with the bug.

**Comment by gornostal** [comment]
Closing as cannot reproduce for now.

**Comment by gornostal** [comment]
Was able to reproduce with GTK 3.22.10 and Xfce on Arch Linux

**Comment by gornostal** [comment]
Fixed in v3.1.1
https://ulauncher.io/#Download
